### DISCUSSION TITLE
Can the renovate the whole version id? How?

### DISCUSSION THREAD

**Discussion by jjcaballero** [renovatebot/renovate#8998] [Request Help]
Let's put the following scenario:

Renovate created an MR with a patch update, for the sake of presentation, let's say version 1.0.1. So, the default branch will be "renovate/dependency-1.0.x". The tests failed and developers are working on the branch to fix those errors.
Once, they are done. They need to wait for approval to merge that version 1.0.1. The approval will take a while.

In the meantime, there is a new version 1.0.2. Renovate does not create a new MR because there are changes on "renovate/dependency-1.0.x" by other authors. 

Is it possible to have Renovate create a separate new MR for the new version?

I assume I could use the _branchTopic_ parameter to make the renovate branch name to contain the whole version? Is this 
possible? 
I have tried this without luck:
`"{{{depNameSanitized}}}-{{{newMajor}}}{{#if isPatch}}.{{{newMinor}}}{{/if}}.{{{newPatch}}}{{#if isLockfileUpdate}}-lockfile{{/if}}"
`
We are using pep04 and 4-digit-version. I wonder if we could use a keyword extracted from another parameter (e.g.: packageRules.versioning) to be used in the _branchTopic_ parameter.

Any advice will be welcome. Also, probably, there is a different approach to accomplish this instead of adjusting the _branchTopic_ parameter.

Thanks!

**Comment by rarkins** [comment]
I can't think of any elegant way to achieve "create a separate branch but only if the existing one is blocked by commits". I'm also not sure why you'd want to get 1.0.2 if you're still fixing 1.0.1, although maybe it's because they're considered separate release streams?

What you probably would need is https://github.com/renovatebot/renovate/issues/4860 first and then adding a feature `separateMultiplePatch` (no feature request exists for that which I'm aware of).
